docs/models/peering/template.md

```diff
@@ -0,0 +1,26 @@
+# Template
+
+A template is a text that can be parsed using a context processor that will
+give a rendered version of the text by completing variable references and
+executing control statements.
+
+Inside Peering Manager, templates use the
+[Jinja2](https://palletsprojects.com/p/jinja/) syntax which allows for complex
+logics building. Depending if the template is for a configuration or an e-mail
+different variables will be exposed to the Jinja2 processor.
```

Comment:
For second sentence, maybe try:
"Different variables are exposed to the Jinja2 processor for email and configuration templates"